tapa_graphir/BUILD.bazel

```diff
@@ -0,0 +1,24 @@
+"""TAPA graphir dependencies."""
+
+# Copyright (c) 2025 RapidStream Design Automation, Inc. and contributors.
+# All rights reserved. The contributor(s) of this file has/have agreed to the
+# RapidStream Contributor License Agreement.
+
+load("@rules_python//python:defs.bzl", "py_library")
+load("@tapa_deps//:requirements.bzl", "requirement")
+
+py_library(
+    name = "tapa_graphir",
+    srcs = glob([
+        "**/*.py",
+    ]),
+    visibility = [
+        "//tapa:__subpackages__",
+        "//tapa_graphir:__subpackages__",
+    ],
+    deps = [
+        requirement(
+            "pydantic",
+        ),
```

Comment:
I think you also need `requirment("pyslang")` due to `tapa_graphir/third_party/systemverilog/const.py`.

Also, prefer to put `requirement("pydantic"),` in a single line.